cmd/utils/flags.go

```diff
@@ -268,6 +270,17 @@ var (
 		Usage:    "Manually specify the Verkle fork timestamp, overriding the bundled setting",
 		Category: flags.EthCategory,
 	}
+	PathBasedSchemeFlag = &cli.BoolFlag{
```

Comment:
Since we'll probably have to live with this flag for the switchover period with people might be wanting to run one or the other explicitly, maybe it would be better to have something similar to db.engine = leveldb/pebble. E.g. trie.schema = hash/path. That way us flipping the default from one to the other will not influence people running with specified flags.

Reply:
Another thing to maybe consider is whether to namespace it as "trie" or "state". The latter is kind of more user friendly (i.e. you don't really want to know what a trie is when wanting to set how many blocks of history to retain, state might be more meaningful). State is also the namespace used in the configs I think.

Reply:
Sounds good to me, I will go with state